docs/en/observability/ci-cd-observability.asciidoc

```diff
@@ -150,6 +150,79 @@ To investigate further, you can view the details of the build captured as labels
 .Contextual attributes of a Jenkins pipeline execution in Elastic Observability
 image::images/jenkins-pipeline-context.png[Attributes of a Jenkins pipeline execution]
 
+[discrete]
+[[ci-cd-store-jenkins-logs]]
+== Store Jenkins pipeline logs in Elastic
+
+Jenkins pipeline build logs can be sent through OpenTelemetry Protocol (OTLP) to be stored
+in an observability backend alongside the traces of the pipeline builds and Jenkins health metrics.
+Storing logs in an observability backend provides several benefits including:
+
+* Better observability, monitoring, alerting, and troubleshooting of the Jenkins instance
+due to the unification of all the signals in the observability backend.
+* Better traceability and control of long term storage for builds so you can more easily
+audit the Software Delivery Lifecycle.
+* Better scalability and reliability of Jenkins by greatly reducing the quantity of data stored in
+Jenkins and limiting the well known file system performance of Jenkins when storing a large history of builds.
+
+[discrete]
+[[ci-cd-store-jenkins-logs-requirements-configuration]]
+=== Requirements and configuration
+
+Storing Jenkins pipeline logs in Elastic requires:
+
+* Elastic Observability version 8.1 or higher.
+* The OpenTelemetry Protocol endpoint configured on the Jenkins OpenTelemetry integrationmust be
+reachable from the Jenkins Agents (i.e. don't specify a localhost OTLP endpoint unless OpenTelemetry
+collectors are also deployed on the Jenkins Agents).
+* When using OpenTelemetry Collectors, set up a logs pipeline in addition to the traces and metrics pipelines.
```

Comment:
```suggestion
* When using OpenTelemetry Collectors, set up a [logs pipeline](https://opentelemetry.io/docs/collector/configuration/#service) in addition to the traces and metrics pipelines.
```

I added an hyperlink to help readers understand that "logs pipelines" here are concepts of the OpenTelemetry Collector, not related to Jenkins pipelines.